docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,73 @@
+# Release 427 (13 Sep 2023)
+
+## Security
+
+* Added access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## ClickHouse connector
+
+* Add support for pushing down `UPDATE` statements to ClickHouse. ({issue}`16445`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+
+## Druid connector
+
+* Add support for pushing down `UPDATE` statements to Druid. ({issue}`16445`)
+
+## Hive connector
+
+* Add support for pushing down `UPDATE` statements to Hive. ({issue}`16445`)
```

Comment:
To Hive ... Hive is no running when using Trino .. where are we pushing to .. the HMS ?

Reply:
https://github.com/trinodb/trino/pull/16445 - there is no support for Hive in this PR only JDBC connectors:
Ignite
MariaDB
MySql
Oracle
Postgres
Redshift
Singlestore
SqlServer